clang/runtime/dpct-rt/include/dpct/blas_utils.hpp

```diff
@@ -19,8 +19,198 @@
 #include <thread>
 
 namespace dpct {
-
 namespace blas {
+namespace detail {
+template <typename target_t, typename source_t> class parameter_wrapper_base_t {
```

Comment:
```suggestion
template <typename target_element_t, typename source_element_t> class parameter_wrapper_base_t {
```

Reply:
Other type aliases are removed, so keep the name.